packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,51 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (model: ModelData): string[] => {
+	let encoder: string;
+	let features: number;
+	let out_channels: number[];
+
+	if (model.id === "depth-anything/Depth-Anything-V2-Small") {
+		encoder = "vits";
+		features = 64;
+		out_channels = [48, 96, 192, 384];
+	} else if (model.id === "depth-anything/Depth-Anything-V2-Base") {
+		encoder = "vitb";
+		features = 128;
+		out_channels = [96, 192, 384, 768];
+	} else if (model.id === "depth-anything/Depth-Anything-V2-Large") {
+		encoder = "vitl";
+		features = 256;
+		out_channels = [256, 512, 1024, 1024];
+	} else {
+		throw new Error("Unsupported model ID");
```

Comment:
I'm actually not sure what would be the behavior here. It could lead to a HTTP 500 for example if model id is not handled. Maybe best to have default placeholders:

```js
encoder = "<ENCODER>"
features = "<NB_FEATURES>"
out_channels = "<OUT_CHANNELS>"
```